python/paddle/fluid/layers/nn.py

```diff
@@ -5827,6 +5828,53 @@ def rank_loss(label, left, right, name=None):
     return out
 
 
+def margin_rank_loss(label, left, right, margin=0.1, name=None):
+    """
+    Margin Rank loss layer for rank problem, which comparing left value and right value be passed in.
```

Comment:
` Margin Rank loss layer for rank problem` -> ` Margin Ranking Loss Layer for ranking problem`
`comparing` -> `compares`
`value` -> `score`
`be passed in` -> `passed in`